core/src/error/mod.rs

```diff
@@ -583,6 +583,9 @@ pub enum ParseError {
     /// time, there are a set of expressions that can be excluded syntactically. Currently, it's
     /// mostly constants.
     InvalidContract(RawSpan),
+    /// Attempt to specify an import, type of which is not known at the moment of compilation.
+    /// `explicit` determines whether explicit import type annotation was used
```

Comment:
Once again, I fear `type` is a bit ambiguous here (because imports can have a type as well). I suggest we use `format` consistently in the documentation and error reporting instead.